supplement.csv

```diff
@@ -1,7 +1,65 @@
+白内障,/,noun,1908,"Cataract, (醫)瞕, 白內障, 內障眼, 眼珠變質",顏惠慶『英華大辭典』s.v. Pearl-eye,
+白熱,1,adjective,1869,"White heat, 白熱",羅存德『英華字典』s.v. Heat,
+擺,1,noun,1869,"Of o'clock, 擺",羅存德『英華字典』s.v. Pendulum+,
+拜金主義,/,noun,1913,Mammon worship,"A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",no lemma given in Schmidt
+扳手,2,noun,1893,其彈子均由後膛旁孔納入，膛下設木把手，用右手把住扳手，再用左手推挽活木把手向前，則彈子從鎗口而出。,鄭觀應『盛世危言‧火器』,
+班次,?,noun,1848,?,林則徐『請淮廉惠補授騰越鎮中軍游擊折』,
+斑馬,/,noun,1899,Zebra,鄺其照『華英字典集成』,no lemma quoted in Schmidt
+版畫,/,noun,1908,"An impression from an engraved plate, block of wood, or other material, 刻版印刷物, 版畫, 印畫",顏惠慶『英華大辭典』s.v. Engraving+,
+版權,/,noun,1887,凡以著作及翻译之图书刻版者，许于三十年间他人不得盗卖，名为版权。,黃遵憲『日本國志』卷十六,Huang dates this source to 1890 but in the database 『日本國志』 is dated to 1887. Schmidt also cites 『日本國志』學術志一
+半旗,/,noun,1911,"At half-mast, 半旗, 半桿旗",衛禮賢『德英華文科學字典』s.v. Halbmast,
+伴侶,/,noun,1848,"Companion, 伴侶",麥都思『英華字典』s.v. Companion+,
+幫,/,noun,1872,"Party or company, 羣黨, 朋黨, 班, 幫",盧公明『英華萃林韻府』s.v. Party,
+綁票,/,verb,1913,"Kidnap, randsom demand, seize for ransom","A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",no lemma quoted in Schmidt
+保留,1,verb,1908,"To hold or retain, 保有, 保留",顏惠慶『英華大辭典』s.v. Keep+,
+保險公司,/,noun,1869,"Insurance company, 保險會, 燕梳會, 擔保會, 保險公司",羅存德『英華字典』s.v. Insurance+,
+保證書,/,noun,1881,?,黃遵憲『日本國志』刑法志二,
+報告,?,noun,1881,?,黃遵憲『日本國志』刑法志一,
+暴風,/,noun,1822,"Tempest of wind, 暴風",R. Morrison『英華字典』s.v. Tempest+,
+暴力,/,noun,1908,"Power exerted against will or consent, 抗願之力, 逼迫之力, 壓力, 暴力, 暴行",顏惠慶『英華大辭典』s.v. Force+,
+暴雨,/,noun,1857,?,A. Wylie『六合叢談』,
+北極星,1,proper noun,1815,"A constellation, 北極星","R. Morrison, _A Dictionary of the Chinese Language in three parts_",
+備考,/,verb,1865,"備考, to retain for future investigation,—said of literary subjects",R. Morrison『五車韻府』s.v. 𤰇/備/僃/偹,
+背景,?,noun,1913,Background,"A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",no lemma quoted in Schmidt
+背面,/,noun,1822,"Inside or back surface, 背面",R. Morrison『英華字典』s.v. Outside,
+被告,/,noun,1703,?,萬濟國『華語官話語法』,
+被面,/,noun,1848,"Coverlet, 被窩, 被面, 被禪, 裨臂, 蓋被, 衾裯, 紟, 袣, 芾, 幭, 披單, 單紟",麥都思『英華字典』s.v. Coverlet,
+被殺,/,preposition + verb,1857,?,A. Wylie『六合叢談』,
+本能,/,noun,1636,各独難普本能、彼此互合、乃易顕舌,陽瑪諾『聖經直解』,
+本體,/,noun,1844,"Essence, 精, 本體, 精氣",衛三畏『英華韵府歷階』s.v. Essence,
+本土,?,noun,1857,?,A. Wylie『六合叢談』,
+本文,?,noun,1857,?,A. Wylie『六合叢談』,
+本義,/,noun,1869,"The proper sense of a word, 本意, 本義",羅存德『英華字典』s.v. Proper,
+本質,/,noun,1672,夫風之本質乃地所発乾熱之気有多端可証一試,南懐仁『坤輿圖説』,
+鼻音,/,noun,1869,"Ditto or nasal sound, 鼻音, 鼻聲",羅存德『英華字典』s.v. Twang+,
+比喻,/,noun,1783,?,萬濟國『華語官話語法』,
+比重,1,noun,1869,"Specific gravity, 比重",羅存德『英華字典』s.v. Gravity+,
+筆名,/,noun,1908,"A false name, 偽名, 冒名, 假名, 別名, 筆名",顏惠慶『英華大辭典』s.v. Pseudonym,
+必然,/,adverb,1703,?,萬濟國『華語官話語法』,
+編者,/,noun,1857,?,A. Wylie『六合叢談』,
+變態,1,noun,1885,常恐後人體，變態猶未盡,黃遵憲『雜感·其三』,
+便秘,/,noun,1844,"Constipation, 大便秘, 閉恭",W. Williams『英華韵府歷階』,
+標本,/,noun,1902,?,梁啓超『飲冰室合集』,
+標記,/,noun,1857,?,A. Wylie『六合叢談』,
+標準語,/,noun,1916,"~ language, 標準語",赫美玲『官話』s.v. Standard,
```

Comment:
```suggestion
標準語,/,noun,1916,"Standard language, 標準語",赫美玲『官話』s.v. Standard,
```
Yeah, we’re not so short of space we have to resort to `~`. Write them out for better UX.

Is that an uppercase or lowercase `S` in the source?

Reply:
It was a ~ in the source. I capitalised the first letters in the quote column. Would you like me to keep the capitalisation of the source?

Reply:
Ha, Schmidt always surprises!

No, keep it that way. I meant to leave Friday for you to look up all the missing/doubtful citations. So verify it in the original source tomorrow.

Reply:
Just to confirm, should I keep the ~ in? And I should keep capitalising the first letters of the quotes?

Reply:
Good for you! Pls list the sources you’ve personally verify in the description of the PR. This will tell others they don’t need to do that again.

Right, keep whatever you saw there.

Since you’ve done 英華字典, tomorrow you’ll be relying mainly on Google Books for the other citations.